packages/flutter/lib/src/widgets/raw_radio.dart

```diff
@@ -113,24 +79,14 @@ class RawRadio<T> extends StatefulWidget {
   /// Set to true if this radio button is allowed to be returned to an
   /// indeterminate state by selecting it again when selected.
   ///
-  /// To indicate returning to an indeterminate state, [onChanged] will be
-  /// called with null.
-  ///
-  /// If true, [onChanged] is called with [value] when selected while
-  /// [groupValue] != [value], and with null when selected again while
-  /// [groupValue] == [value].
-  ///
-  /// If false, [onChanged] will be called with [value] when it is selected
-  /// while [groupValue] != [value], and only by selecting another radio button
-  /// in the group (i.e. changing the value of [groupValue]) can this radio
-  /// button be unselected.
+  /// To indicate returning to an indeterminate state.
```

Comment:
Did you mean to remove this block? The remaining sentence doesn't make sense by itself.